### ISSUE TITLE
The Game Crashed with buying in the shop

### ISSUE THREAD

**Issue by KirbyTailZ** [WagicProject/wagic#1056]
Hello, I am a Wagic player since its official versions and seeing that you are releasing new versions with improvements, I decided to download the latest version and I have a significant problem.

The minor problem is that the card downloader that you provide does not work, each card that starts to download fails, I don't know the reason, I am sorry not to be able to play with all the cards with images but it is minimal.

The serious thing is in the store, when I'm buying cards, I don't know the reason, when I buy infrequent it crashes, and curiously, it only happens with this rarity and I explain myself.

I buy envelopes and I don't crash, I usually buy common cards, 4 copies, so many copies, and nothing, with the rare ones I have bought a pair and it hasn't crashed, but when I want to buy an infrequent one, ruinampago type, or similar, it freezes me the game and it closes ... I don't know why it happens.

I use windows 10 pro, I have an AMD Ryzen 5 2600 computer at 3.4 ghz, 16 gb ram DDR4, Nvidia GeForce 1650 ti GDDR5 4 GB.

If you can tell me what is causing this failure to occur, since the downloader commented on it in case I was doing something wrong, but the most important thing for me is to solve the store, because it makes it impossible for me to play, and how to start the game with one copy per card is a bit difficult for me to play, so I like to buy some cards to start creating my first deck, thank you very much!

**Comment by Vitty85** [comment]
Hi,

did you follow the instruction for Downloader? You have to copy the library of Java security in the right path.

btw you can download the full packages of Wagic from official forum:

http://wololo.net/forum/viewtopic.php?f=17&t=66595&start=20#p99279

See you.

**Comment by Vitty85** [comment]
Hi,

I tried top buy both Blightning  and Path To Exile and i didn't have any crash (i used Android and Windows version), i guess it's not about that or it's a local problem to your installation, try with the full packages from forum as i said you before. Meanwhile i close this issue.

**Comment by KirbyTailZ** [comment]
Hello, I downloaded it from the forum yesterday and I was testing it today and the same thing happens to me again, this time it has been with double lands, I do not know the reason for those failures, it has been ocn the Arcane Sanctuary (the triple ground) and the rare land sejiri (enter tapped, gain 1 life and add blue or white) crashes again ... and is downloaded from the forum :(

It is a strange failure is that I do not know where u_u can come from

Thanks anyway for the help because I don't know if it will be a casual error that you are happening with my computer

a greeting

**Comment by Vitty85** [comment]
Are you using a theme? Maybe it can depends on that.

Or it can depends form language, maybe there are bad chars in Spanish language file, I’m using English file with MTG theme.

See you.

--
Inviato da iPhone

**Comment by KirbyTailZ** [comment]
Yes, I think that may be the solution, the theme I have not touched since I like the new one that you added is really nice, I tried to change the language, I had it in Spanish, and I passed it to English, that the magic letters in English I know them really well, and at the moment it has not crashed even once, since the last crash was buying an envelope of MB1 something that already seemed extremely strange to me, I changed it to English, and bought 2 envelopes and some infrequent letter and it has not given me any other failure.

So surely, the error has to come from the Spanish language, some piece of the code that does not work well and causes the game to crash when buying a product in the store.

Thank you very much for everything :) I am delighted to be able to play, the changes you have made are huge, it looks really good and it goes even more fluid than the official version I played.

Greetings!
